src/main.rs

```diff
@@ -0,0 +1,123 @@
+use anyhow::{bail, Error};
+use structopt::StructOpt;
+use wasi_cap_std_sync::WasiCtxBuilder;
+use wasi_experimental_http_wasmtime::HttpCtx;
+use wasmtime::{Func, Instance, Linker, Store, Val, ValType};
+use wasmtime_wasi::Wasi;
+
+#[derive(Debug, StructOpt)]
+#[structopt(name = "wasi-experimental-http-runner")]
+struct Opt {
+    #[structopt()]
```

Comment:
Is this missing a `long` definition?

Reply:
The top-level command?
It's the only command, is it really necessary?